src/store/actions/set_active_item.js

```diff
@@ -25,5 +25,18 @@ export default firestoreAction(async ({ bindFirestoreRef, unbindFirestoreRef, co
   const kpisRef = db.collection('kpis').where('parent', '==', item);
   await bindFirestoreRef('kpis', kpisRef, { maxRefDepth: 0 });
 
+  const products = await db
+        .collection('products')
+        .where('department', '==', item)
+        .limit(10)
+        .get()
+        .then((snapshot) => snapshot.docs.map((doc) => doc.ref));
+
+  const subKpisRef = db
```

Comment:
Med dette laster vi vel inn "subKpis" globalt når man navigerer i OKR-trackeren, selv om disse kun er relevante for dashboardet?

Reply:
Det stemmer sikkert, har ikke helt skjønt hvordan disse actions-tingene funker enda. Skal teste litt...